hr_employee_phone_extension/models/__init__.py

```diff
@@ -1,3 +1,5 @@
-#! -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
+# ©  2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
No need to add copyright in `__init__.py` files as long as it contains no code and just imports.